vuepress/docs/next/docs/curate/ecr-bundle-details.md

```diff
@@ -239,9 +240,11 @@ Here is an example of a Page Template descriptor:
       </html>
 
 ## Page
-This descriptor enables a page to be created and published via a bundle. Page status can be `published` or `draft`. The widget section can be used to fully configure a page layout.
+This descriptor enables a page to be created and published via a bundle. Page status can be `published` or `draft`. The Widget section can be used to fully configure a page layout. 
 
-**Page descriptor.yaml.**
+Groups in a page descriptor are configured by `ownerGroup` and `joinGroups`. The `ownerGroup` property assigns the group of users who can manage the entity in the AppBuilder. The `joinGroups` property defines who can view or access the entity. A group designation of "free" signifies a public entity.
```

Comment:
this part is confusing>> "The `ownerGroup` property assigns the group of users who can manage the entity in the AppBuilder. The `joinGroups` property defines who can view or access the entity." Who is assigning what to whom? Isn't it more about if you assign a user to 'ownerGroup', they can manage the entity in the AppBuilder... same with joinGroups defining who can view or access? I think property is misleading maybe.

Reply:
Maybe "assigns" -> "specifies". And then -> `joinGroups` specifies who can view or access the page. For example, setting the `ownerGroup` to `free` means anyone with access to the AppBuilder can manage the page, whereas setting the `joinGroup` to `free` means any end user can see view the page in the application.

Reply:
this is always subtle and tricky for me, because someone determines what/who a property or attribute consists of, but in turn that property/attribute controls the actions or access of what/who... agree 100% wrt "assigns" here tho

Reply:
I know in a broad sense, these can be viewed as properties or attributes but in these specific cases, aren't these groups and roles and they provide the attribute of who has what access

Reply:
actually groups and roles are properties... properly identifying programming elements is not a strength of mine, but nathan defined them as such so i'm confident :P regardless i think your point of who is assigning what to whom is the important bit, like is the programming element assigning something to a user or a user being assigned to a programming element. sometimes it's obv and cut and dried, but other times there's some gray area. in this case we confirmed the properties aren't assigning anything, just specifying; an admin is assigning a value to the property that defines user access, etc.